templates/zigbee2mqtt.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>Zigbee2MQTT</Name>
+  <Repository>koenkk/zigbee2mqtt</Repository>
+  <Registry>https://hub.docker.com/r/koenkk/zigbee2mqtt/</Registry>
+  <Network>bridge</Network>
+  <Privileged>false</Privileged>
+  <Support>https://www.zigbee2mqtt.io/information/docker.html</Support>
+  <Project/>
+  <Overview>Zigbee &#x1F41D; to MQTT bridge &#x1F309; using zigbee-shepherd &#x1F415;   Converted By Community Applications   Always verify this template (and values) against the dockerhub support page for the container</Overview>
+  <Category/>
```

Comment:
Would be nice is this was filled
[link](https://selfhosters.net/docker/templating/templating/#21-categories)